### ISSUE TITLE
Could the README mention how to fire this up locally?

### ISSUE THREAD

**Issue by joshgoebel** [highlightjs/hljs_org#3]
Is there some easy way to fire this up locally to test changes?  Surely there is.

Could we add a tiny section on that?

**Comment by allejo** [comment]
This gets the website running, but I haven't looked into getting a hljs build incorporated with it

```bash
# Requires Python 3.9 (I think)
brew install libmemcached

virtualenv venv
source venv/bin/activate
pip install -r requirements.txt

# You need to setup a local PostgreSQL server on your own
# Create a `hljs_org` database in that Postgres server

python manage.py migrate
python manage.py runserver
```

**Comment by isagalaev** [comment]
Fixed in ea791e2.
